lib/omniauth/strategies/openid_connect.rb

```diff
@@ -42,7 +43,6 @@ class OpenIDConnect
       option :send_nonce, true
       option :send_scope_to_token_endpoint, true
       option :client_auth_method
-      option :post_logout_redirect_uri
```

Comment:
I think create a breaking change for this is useless. What's about keeping both with a deprecation warning on this option ?